ponyo/vae.py

```diff
@@ -344,11 +324,11 @@ def tybalt_2layer_model(
         validation_frac,
     )
 
-    encoded_rnaseq_df = encoder.predict_on_batch(rnaseq)
-    encoded_rnaseq_df = pd.DataFrame(encoded_rnaseq_df, index=rnaseq.index)
+    encoded_expression_df = encoder.predict_on_batch(expression_data)
```

Comment:
ahh i see, bc it could be any expression data, not sure rnaseq?

Reply:
Yes, I did a little clean up here to try to make the names of the variables more descriptive